content/en/docs/reference/glossary/host-aliases.md

```diff
@@ -14,4 +14,4 @@ tags:
 
 <!--more-->
 
-[HostAliases](/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-corev) is an optional list of hostnames and IP addresses that will be injected into the Pod's hosts file if specified. This is only valid for non-hostNetwork Pods.
+[HostAliases](https://kubernetes.io/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#hostalias-v1-core) is an optional list of hostnames and IP addresses that will be injected into the Pod's hosts file if specified. This is only valid for non-hostNetwork Pods.
```

Comment:
@davivcgarcia , What do you think about using,
/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#hostalias-v1-core

Reply:
Oh my! Sorry my mistake, I don't know why I replaced it with the full URL. I'll fix that right away!